test/Test/Extensions/Parser.hs

```diff
@@ -238,6 +239,43 @@ cppSpec = describe "Parsing extensions with CPP" $ do
         ])
         [TypeApplications, LambdaCase]
 
+optionsGhcSpec :: Spec
+optionsGhcSpec = describe "Parsing LANGUAGE and OPTIONS_GHC pragmas" $ do
+    itShouldParse
+        "{-# OPTIONS_GHC -fno-warn-orphans #-}"
+        []
+    itShouldParse
+        "{-# options_ghc -freverse-errors #-}"
+        []
+    itShouldParse
+        "{-# OPTIONS_GHC -fno-warn-orphans -freverse-errors #-}"
+        []
```

Comment:
Let's see if this will work as well :wink: 
```suggestion
        []
    itShouldParse
        [ "{-# OPTIONS_GHC"
        , " -fno-warn-orphans"
        , " -freverse-errors "
        , "#-}"
        []
```